config/config.exs

```diff
@@ -142,9 +142,8 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
-# "code" is the secret value returned by AWS to /auth/cognito/callback
 log_filter_params =
-  ~w(password code token guardian_default_claims guardian_default_resource guardian_default_token)
+  ~w(password token guardian_default_claims guardian_default_resource guardian_default_token)
```

Comment:
I'm unsure about this - perhaps we should keep `code` filtered?

Reply:
I _think_ this is okay? I can't see a variable in the keycloak response that's simlar to the code variable in the cognito response. It doesn't really hurt to keep though it seems?

I'm good either way I think